src/utils/externalLink.ts

```diff
@@ -0,0 +1,7 @@
+// Adds 'https://' to the start of a link if it doesn't already have it
+export function withHttps(link: string | undefined) {
+  if (link?.slice(0, 8) !== 'https://' && link?.slice(0, 7) !== 'http://') {
```

Comment:
I think you'll need to test for `link` not being undefined first. I think if it is, you'll always end up with `https://undefined`

Reply:
Great pickup, fixed